src/System.Management.Automation/engine/InternalCommands.cs

```diff
@@ -2717,6 +2718,42 @@ protected override void EndProcessing()
             Context.EngineSessionState.CurrentScope.StrictModeVersion = _version;
         }
     }
+
+    /// <summary>
+    /// Provides argument completion for StrictMode Version parameter.
+    /// </summary>
+    public class StrictModeVersionArgumentCompleter : IArgumentCompleter
+    {
+        private static readonly string[] s_StrictModeVersions = new string[] { "1.0", "2.0", "3.0", "Latest" };
```

Comment:
Maybe reverse order?

Reply:
@iSazonov That seems much nicer, easier to get `Latest` as first completion. Reversed in 7995d0cbd.